development-setup-wizard.py

```diff
@@ -0,0 +1,244 @@
+            f'it looks like "${dir_name}" is not empty,'
+            'do you want to blow it away and start over? [y/n] ']))
+        if answer.lower() == 'y':
+            shutil.rmtree(dir_name)
+
+    hooray(f'Checking of {dir_name} completed')
+
+
+def docker_compose_up(args):
+    print(bold('(Re)building docker containers'))
+    result = subprocess.run(['docker', 'compose', 'up', '--build', '-d'])
+    if result.returncode != 0:
+        die('failed to start docker containers')
+    hooray('Successfully built and started docker containers')
+
+
+def set_env_values(args):
+    if os.environ.get('CAMPHORIC_SETUP_WIZARD_RECREATE_SECRET_KEY') == 'no':
+        return
+
+    filename = './env/local/django.env'
+    contents = ''
+    with open(filename) as f:
+        contents = f.read().strip()
+
+    current_key_pair = ''
+    for line in re.split(r'[\r\n]+', contents):
+        if line.startswith('SECRET_KEY='):
+            current_key_pair = line
+            break
+
+    current_key = ''
+    if current_key_pair:
+        current_key = current_key_pair.split('=')[1]
+    else:
+        current_key = 'your-django-secret-key-here'
+        contents += '\nSECRET_KEY=your-django-secret-key-here'
+
+    if args.preserve:
+        return
+
+    if (current_key != 'your-django-secret-key-here'
+            and 'CAMPHORIC_SETUP_WIZARD_RECREATE_SECRET_KEY' not in os.environ):
+        answer = input(' '.join([
+            f'it looks like there is already a SECRET_KEY in your {filename} file.',
+            'Do you want replace it with a newly generated one? [y/n] ']))
+        if answer.lower() != 'y':
+            return
+
+    secret_key = get_secret_key()
+    contents = re.sub(r'SECRET_KEY=.*', f'SECRET_KEY={secret_key}', contents)
+    with open(filename, 'w') as f:
+        f.write(contents)
+    hooray(f'Successfully wrote secret key to {filename}')
+
+    # Rebuild if we added secret key
+    docker_compose_up(args)
+
+
+def get_secret_key():
+    cmd = ' '.join([
+        'docker compose exec -T django python -c',
+        '"from django.core.management.utils import get_random_secret_key;',
+        'print(get_random_secret_key())"'])
+    result = subprocess.run(cmd, shell=True, capture_output=True)
+    secret_key = (
+        result.stdout.decode().strip()
+        .replace("'", "\\'")
+        .replace("$", "_"))  # django-environ uses $ for interpolation
+
+    if not secret_key:
+        die(
+            "Could not successfully get secret key",
+            f"tried '{cmd}'"
+            f'\n{result.stderr.decode()}')
+    else:
+        hooray('Successfully got secret key')
+
+    return secret_key
+
+
+def create_django_superuser(args):
+    print(bold('Create your Django superuser:'))
+    subprocess.run([
+        'docker', 'compose', 'exec', 'django',
+        'python', 'manage.py', 'createsuperuser', '--noinput'])
+
+
+def load_sample_data(args):
+    print(bold('Loading Lark Camp sample data...'))
```

Comment:
This now adds more than Lark Camp Data - perhaps change this to just "Loading sample data"